ui/jest.config.js

```diff
@@ -1,9 +1,13 @@
 module.exports = {
   preset: "ts-jest",
+  testEnvironment: "jsdom",
+  setupFilesAfterEnv: ["<rootDir>/setupTests.ts"],
   transformIgnorePatterns: [
     "node_modules/(?!(argo-ui)/)"
   ],
   moduleNameMapper: {
-    "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js"
+    "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js",
+    "\\.(jpg|jpeg|png|gif|webp|svg)$": "<rootDir>/__mocks__/fileMock.js"
```

Comment:
Not sure why this was changed, but we aren't importing images in the tests, and `fileMock.js` doesn't exist in this repository, so I think this should be reverted:
```suggestion
    "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js"
```